### ISSUE TITLE
Bower dependencies are not resolved correctly

### ISSUE THREAD

**Issue by PandaWhisperer** [smies/sails-generate-bower#5]
I used this generator in order to install bootstrap in my Sails app. Bootstrap depends on jQuery, so it automatically gets installed as dependency. However, the sails-linker doesn't seem to know about the bower dependency tree, so it ends up including the scripts in alphabetical order, placing jQuery _after_ bootstrap. When I load my app in the browser, I get the error message

```
Uncaught TypeError: $ is not a function
```

as you'd expect.

**Comment by narangrishab-cmu** [comment]
@PandaWhisperer did you find a fix for this?

**Comment by PandaWhisperer** [comment]
@narangrishab-cmu nope. But I haven't really done any JS backend development in the last few years anyways. 

Since bower has been deprecated widely in favor of webpack, I imagine this package isn't really relevant anymore, anyways.
